website/screens/components/tag/usage/examples/iconUsage.ts

```diff
@@ -17,15 +17,15 @@ const code = `() => {
   return (
     <DxcInset space="2rem">
       <DxcRow justify="spaceEvenly">
-        <DxcStack gutter="large">
```

Comment:
Why should I add a gutter to the `DxcRow`?

Reply:
Because it is shown like this and we can take advantage of this PR to change it:

![image](https://user-images.githubusercontent.com/10975289/178249519-376fd67d-11dd-444d-8f61-d332a65402a8.png)

Reply:
Done!